autogen/convert_constants.py

```
@@ -200,6 +206,14 @@ def process_file(filename):
                 constants.extend(block['else'])
         else:
             c = process_line(filename, line, block_stack is not None)
+            global enum_constants, enum_entries
+            if enum_entries > 0:
+                if c is not None:
+                    enum_constants['constants'].append(c)
+                enum_entries -= 1
+                if enum_entries == 0:
+                    c = enum_constants
+                else: continue
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)